cli/src/main.rs

```diff
@@ -484,6 +485,20 @@ impl World for SystemWorld {
             .get_or_init(|| read(path).map(Buffer::from))
             .clone()
     }
+
+    fn today(&self, offset: Option<i64>) -> Option<(i32, u8, u8)> {
+        let datetime = match offset {
+            None => chrono::Local::now().naive_local(),
+            Some(o) => (chrono::Utc::now() + chrono::Duration::hours(o)).naive_utc(),
+        };
+
+        // Month/day are always in range of u8
+        Some((
+            datetime.year(),
+            datetime.month().try_into().unwrap(),
+            datetime.day().try_into().unwrap(),
+        ))
```

Comment:
this should do the actual chrono call only once per compilation. otherwise, you might get weird results if your compilation continues across the date boundary.

Reply:
Hmm, not sure I follow. Do you mean I should only have one call to chrono in the whole function instead of calling it once for `None` and once for `Some`?

Reply:
I mean that the world should have a field (probably `Cell<Option<..>>` that caches the date and is reset to `None` at the start of the next compilation (in watch mode). The function could be called multiple times after all.

Reply:
I'll look into it. :)